src/komponenter/utvandret/oppgave-opprettet.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { Feilmelding, Undertittel } from 'nav-frontend-typografi';
+import Hjelpetekst from 'nav-frontend-hjelpetekst';
+import Veilederpanel from 'nav-frontend-veilederpanel';
+import Alertstripe from 'nav-frontend-alertstriper';
+import virkedager from '@alheimsins/virkedager';
+import { Kontaktinformasjon } from '../oppholdstillatelse';
```

Comment:
Mye her er veldig likt som for ugyldig opphold. Jeg er usikker på om jeg skal duplisere eller lage felles komponenter/styling. Hva tror dere er best?

Reply:
Er det ikke når du gjør det for 3. gang du lager komponent ;-)
Ville ventet litt og sett an. Det er er kanskje på vei mot en egen kontakt-meg-bruker komponent? Enn så lenge så er det så lite kode at jeg ikke helt ser verdien i gjenbruket.